ui/main/src/app/modules/feedconfiguration/feedconfiguration.component.ts

```diff
@@ -84,8 +87,23 @@ export class FeedconfigurationComponent implements OnInit {
 
     private makeProcessesWithoutGroup() {
         this.processesDefinition.forEach(process => {
-            if (! this.findInProcessGroups(process.id))
-                this.processesWithoutGroup.push(process.id);
+            if (! this.findInProcessGroups(process.id)) {
+                let processLabel = (!!process.name) ? this.getI18nPrefixFromProcess(process) + process.name :
+                    this.getI18nPrefixFromProcess(process) + process.id;
+
+                this.translateService.get(processLabel).subscribe(translate => { processLabel = translate; });
+                this.processesWithoutGroup.push({idProcess: process.id,
+                                                 processLabel: processLabel});
+            }
+        });
+        this.processesWithoutGroup.sort((obj1, obj2) => {
+            if (obj1.processLabel > obj2.processLabel) {
+                return 1;
+            }
+            if (obj1.processLabel < obj2.processLabel) {
+                return -1;
+            }
+            return 0;
```

Comment:
Avoid duplication : Create a method sortByAlphabeticalOrder and send this method as parameter for processesWithoutGroup.sort

Reply:
Done.